projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -247,3 +246,13 @@ def convert_job(job_dir: str):
         }
         processor.cleanup()
         return message
+
+
+def expand_path(path: str):
+    """
+    Expands the path parameter to be rooted at the file system location where Jupyter was initially ran.
+
+    :param path: path parameter provided by the user
+    :return: expanded path
+    """
+    return pathlib.Path(os.getcwd()).joinpath(path)
```

Comment:
What happens if the user directly puts the full path, not relative Jupyter path?

Reply:
it would fail

Reply:
@gageorgiev don't u think it would be nice to cover both cases ?

Reply:
My understanding of the ticket was that we should want that to fail:
```
dditionally, the path input should be fixed so that it always points relatively to the path assumed root by the Jupyter server, i.e. the directory is was ran inside of. This means that users should always provide the full path starting from the root Jupyter dir...
```

Reply:
I suppose it is all good then but what kind of a message will the user get if it fails like that? both the strings concatenated and saying it is not existing? I think it would look strange

Reply:
@gageorgiev I see, maybe then we should change the labels of all path inputs to explain more that it only accepts jupyter relative paths?